lib/elixir/lib/io/ansi.ex

```diff
@@ -170,6 +170,29 @@ defmodule IO.ANSI do
   @doc "Sends cursor home."
   defsequence(:home, "", "H")
 
+  @doc """
+  Sends cursor to the absolute position specified by `lines`, `columns`, 
+  with 0,0 being the top left corner.
```

Comment:
The first line of the documentation should be a summary of the function. Let's reshape this to be:

```elixir
@doc """
Sends cursor to the absolute position specified by `line` and `column`.

Line `0` and column `0` would mean the top left corner.
"""
```